src/block-management/inline-toolbar/index.js

```diff
@@ -52,18 +52,31 @@ export default class InlineToolbar extends React.Component<PropsType> {
 	}
 
 	render() {
+		const { rowIndex } = this.props;
+		const rowNumber = rowIndex + 1;
+		let moveUpButtonTitle = `Move up from row ${ rowNumber }`;
```

Comment:
Same here, if we can use internationalization, it would be nice :)

Reply:
These are internationalized using `__` in the props passed to the `ToolbarButton` is that ok?

Reply:
Yes, to be clear you need constant strings here otherwise it won't work:
```
let moveUpButtonTitle = sprintf( __( 'Move up from row %d' ), rowNumber );
```

Reply:
Yeah I understand now, thanks for explaining that in the previous comment!